File: editorial/chap2-2/prob2-2-4.cpp
#include <iostream>
using namespace std;

int N, A[109];
int Answer = 0;

int main() {
	// 入力
	cin >> N;
	for (int i = 1; i <= N; i++) {
		cin >> A[i];
	}

	// 答えの計算
	for (int i = 1; i <= N; i++) {
		Answer += A[i];
	}

	// 出力
	cout << Answer % 100 << endl;
	return 0;
}

File: editorial/chap2-2/prob2-2-4.java
import java.util.*;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// “ü—Í
		int N = sc.nextInt();
		int[] A = new int[N + 1];
		for (int i = 1; i <= N; i++) {
			A[i] = sc.nextInt();
		}

		// “š‚¦‚ð‹‚ß‚é
		int Answer = 0;
		for (int i = 1; i <= N; i++) {
			Answer += A[i];
		}

		// o—Í
		System.out.println(Answer % 100);
	}
};